src/sandbox/scripts/prepare_vds_for_sites.py

```diff
@@ -0,0 +1,140 @@
+"""Create a dense MatrixTable from a VDS, along with filtering and repartitioning.
+
+This script subsets a given VDS by either a number of samples, a region or set of regions, or both.
+
+Arguments:
+    --vds-path: str. Path to the VDS in GCS.
+    --samples-to-drop: str, optional. Comma separated list of hail tables with samples to drop
+    --exome-intervals: str. The path to the BED file with the exome intervals that you want to use for filtering.
+    --variant-table: str. Path to the hail table that contains variants that you want to filter on
+    --n-partitions: int. The number of partitions to repartition the end MatrixTable into
+    --sites-table-outpath: str. The path to write the resultant MatrixTable
+    --chromosome: str, optional. An optional chromosome to subset to.
+
+Example usage:
+
+analysis-runner --dataset hgdp-1kg \
+    --access-level test \
+    --output-dir test-subset \
+    --description 'Test VDS subsetting script' \
+    python3 prepare_vds_for_sites.py --vds-path gs://cpg-hgdp-1kg-main/vds/v1-0.vds/ \
+    --exome-intervals gs://cpg-common-main/references/exome-probesets/hg38/mackenzie_intersect_exome_regions.bed \
+    --variant-table gs://cpg-common-main/references/gnomad/v4.1/ht/gnomad.genomes.v4.1.sites.ht \
+    --n-partitions 100 \
+    --sites-table-outpath gs://cpg-common-main/references/ancestry/hgdp-1kg-wgs-pruned_variants.ht
```

Comment:
A very minor nitpick - the output will be a .mt
```suggestion
    --sites-table-outpath gs://cpg-common-main/references/ancestry/hgdp-1kg-wgs-pruned_variants.mt
```

Reply:
Oops